webprojects/snippets.md

```diff
@@ -0,0 +1,107 @@
+
+If your language is not listed above and you would like to have a snippet created for special occasion, such as new product or feature launch, a major holiday or campaign, please follow these steps.
+
+### How to create a good snippet?
+
+#### Leverage what is already written and localize it
+
+If creating your own is challenging, feel free to leverage what was created by the snippets team. Browse through [the snippets written this year so far](https://github.com/mozilla-l10n/engagement-l10n/tree/master/en-US/snippets/2017), you may find a message that resonates to your particular market, product and/or feature launch or a campaign.
+
+#### Create your own
+
+Writing snippets takes a lot of effort. Follow [the same guidelines](https://docs.google.com/presentation/d/1-rR32aMK60u6EvHvRCyINI0J_KH0_shbD_jbPqF_VUI/edit#slide=id.g1632693f0a_0_21) that the snippets team adhere to.
+
+Snippets should be to the point. Try writing your snippet then cut it down until you are left with a sentence that is quick, clever, and delightful. A good snippet copy has a few of the following qualities and characteristics:
+
+* Tangible: quality that can be easily measured - readability, length, grammar, CTA (occasional), and support graphic
+    * LENGTH: the ideal snippet is between 80 and 130 characters. Think of a tweet. Omit needless words.
+    * READABILITY: Short concise words worked for Hemingway, they’ll work just fine for you. Keep the 3 syllable words and technical jargon to a minimum.
+    * CTA(Occasional): You don’t always need to blatantly tell people what to do, but sometimes you do.
+    * GRAMMAR: Do your best to catch mistakes, there is a chance that all checkpoints missed will miss it but 100 million people surely won’t.
+    * SUPPORT GRAPHIC: All simple snippet copy is accompanied by an icon. Your icon and copy should work in unison. Select the [icon from this gallery](https://drive.google.com/drive/folders/0Bz48kfsl_32OMkhVakFobXZ2MFk) that best supports the message. Here are some examples of matching the graphics with the tones of the messages.
+
+![Icons](/assets/images/webprojects/snippets/snippet_w_icon_examples.png)
+
+* Intangible: quality that is recognized but difficult to quantify - quick, clever, alert, untamed
+
+    * QUICK: Get to the point with short words and sentences.
+    * CLEVER: Find humor, irony and wordplay. Reference modern life.
+    * ALERT: Ready to offer tips, advice, and guidance like a trained concierge.
+    * UNTAMED: Be quirky. Do and say things no other brand would.
+
+In addition to the DOs and DONTs mentioned above in terms of localizing a snippet, it is very important that this is the result of a team collaborative effort. The proposed snippet should come in both target language and in English so non-speakers understand what is being conveyed.
+
+Do submit multiple variant snippets of the same message, or multiple snippets of different messages to increase your chance of having one approved.
```

Comment:
How about this? `Try to submit multiple variant snippets of the same message, or multiple snippets of different messages if you can, to lower the chances of having some snippets rejected.` Trying to make it sound more encouraging, we may be able to find even better :)